### PR TITLE
Import JS tools from the `documentation` repository

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR updates the `_docs` submodule and cleanups JS imports.

**Files that were imported from the [`documentation`](https://github.com/SpineEventEngine/documentation) repository:**
- Styles, JS, and source files for the architecture diagram;
- Styles and JS for the `code-tabs`.

The duplications from this site have been removed.

**Removed libraries that are not used:**
- `Bootstrap 3`;
- `jquery.visible.min`.

**Updated libraries:**
- `cookie.js`;
- `anchor.min.js`.

### CODE REVIEW

File: libs/anchor.min.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
....icon&&(r.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(r.style.lineHeight="inherit")),"left"===this.options.placement?(r.style.position="absolute",r.style.marginLeft="-1em",r.style.paddingRight=".5em",e[i].insertBefore(r,e[i].firstChild)):(r.style.paddingLeft=".375em",e[i].appendChild(r))}for(i=0;i<p.length;i++)e.splice(p[i]-i,1);return this.elements=this.elements.concat(e),this},this.remove=function(A){for(var e,t,o=w(A),i=0;i<o.length;i++)(t=o[i].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(o[i]))&&this.elements.splice(e,1),o[i].removeChild(t));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var e=document.createElement("textarea");return e.innerHTML=A,A=e.value,this.options.truncate||d(this.options),A.trim().replace(/'/gi,"").replace(/[& +$,:;=?@"#{}|^~[`%!'<>\]./()*\\\n\t\b\v\u00A0]/g,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&-1<(" "+A.firstChild.className+" ").indexOf(" anchorjs-link "),A=A.lastChild&&-1<(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ");return e||A||!1}}});
```
[the remark this answers is not in the document]
Comment (JuliaEvseeva) [reply libs/anchor.min.js:8]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (JuliaEvseeva) [reply libs/anchor.min.js:8]: @sonatype-lift ignore

File: libs/js.cookie.min.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
...mber"==typeof(i=e({},o,i)).expires&&(i.expires=new Date(Date.now()+864e5*i.expires)),i.expires&&(i.expires=i.expires.toUTCString()),t=encodeURIComponent(t).replace(/%(2[346B]|5E|60|7C)/g,decodeURIComponent).replace(/[()]/g,escape);var c="";for(var u in i)i[u]&&(c+="; "+u,!0!==i[u]&&(c+="="+i[u].split(";")[0]));return document.cookie=t+"="+n.write(r,t)+c}}return Object.create({set:r,get:function(e){if("undefined"!=typeof document&&(!arguments.length||e)){for(var t=document.cookie?document.cookie.split("; "):[],o={},r=0;r<t.length;r++){var i=t[r].split("="),c=i.slice(1).join("=");try{var u=decodeURIComponent(i[0]);if(o[u]=n.read(c,u),e===u)break}catch(e){}}return e?o[e]:o}},remove:function(t,n){r(t,"",e({},n,{expires:-1}))},withAttributes:function(n){return t(this.converter,e({},this.attributes,n))},withConverter:function(n){return t(e({},this.converter,n),this.attributes)}},{attributes:{value:Object.freeze(o)},converter:{value:Object.freeze(n)}})}({read:function(e){return'"'===e[0]&&(e=e.slice(1,-1)),e.replace(/(%[\dA-F]{2})+/gi,decodeURIComponent)},write:function(e){return encodeURIComponent(e).replace(/%(2[346BF]|3[AC-F]|40|5[BDE]|60|7[BCD])/g,decodeURIComponent)}},{path:"/"})}));
```
[the remark this answers is not in the document]
Comment (JuliaEvseeva) [reply libs/js.cookie.min.js:2]: @sonatype-lift ignore

**Comment by JuliaEvseeva** [comment]
@armiol PTAL

**Review by armiol — APPROVED** [review]

**Pull request merged**